common/src/arch.h

```diff
@@ -30,23 +30,23 @@
 
 // Architecture include. Use this one instead of arch-<platform>
 
-#if !defined(arch_h)
+#if !defined(DYNINST_host_arch_h)
```

Comment:
This one is a macro guard. It didn't seem to break anything, but let's change it back just in case.

Reply:
@hainest good catch. This definitely should not have ben changed.